docs/JestObjectAPI.md

```diff
@@ -983,6 +983,54 @@ This function is not available when using legacy fake timers implementation.
 
 :::
 
+### `jest.advanceTimersToNextTimerAsync()`
```

Comment:
Could you place each async entry just after its sync counterpart? For instance, this one would follow `jest.advanceTimersToNextTimer`.